src/js/index.js

```diff
@@ -195,26 +197,63 @@ function readBeatsTxt(filename){
 /**
  * Method that converts mp3 to wav
  */
-function ffmpegToWAV(filepath){
+function ffmpegToWAV(file){
+   
+    var filepath = file.path;
 
-    var filepath = window.audioFile.filePath;
-
-    var filename = window.audioFile.fileName;
+    var filename = file.name;
 
     var fileSplit = filename.split('.');
 
     fileSplit.pop();
 
     var fileNameOnly = fileSplit.join(".");
 
-    const convertionCommand = window.ffmpegLocation + " -v quiet -y -i \""  + filepath + "\" -c:a pcm_s16le \"" + window.outputDir + fileNameOnly + ".wav\""
+    var newFileName = fileNameOnly + ".wav";
+
+    var newFilePath = window.outputDir + newFileName;
+
+    // FFmpeg to convert all files to .wav / stereo -> mono
+    const convertionCommand = window.ffmpegLocation + " -v fatal -y -i \""  + filepath + "\" -ac 1 -c:a pcm_s16le \"" + newFilePath + "\"";
+
+    // Call the sync version of exec
+    try {
+        execSync(convertionCommand, {encoding: "UTF-8"});
+    } catch (e) {
+        return null;
+    }
+    
+    // Store the new file parameters
+    var newFile = {
+        name : newFileName,
+        path : newFilePath,
+        treePath : file.treePath,
+        oldName : file.name
+    }
 
-    // Call the sync version
-    execSync(convertionCommand, {encoding: "UTF-8"}, function (err){})
+    return newFile;
 }
 
-function dragOverHandler(ev)
-{
+/*
+ * Method that generates a BLOB from given filepath
+ */
+
+function blobFromPath(path){
+    if (!fs.existsSync(path)){
+        return null;
+    }
+    // Read the file data into a buffer
+    var buff = fs.readFileSync(path);
```

Comment:
Isn’t buff already a Blob here?

Reply:
[readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options) returns a buffer with the file data, it does not act like the `dataTransfer` event and does not produce a `BLOB` or a `File` object with their respective fields (like type etc.)